### PR TITLE
Optimize the getImage function and put into the utils file

### PR DESCRIPTION
**Pull request by MGmadhavan**

### CODE REVIEW

**Review by prasannasurya12 — APPROVED** [review]

File: src/components/Demo.vue [hunk 1]
```diff
@@ -19,15 +19,17 @@
    </div>
 </div>
 
-</div>
```
Comment (prasannasurya12) [inline src/components/Demo.vue:22]: ok

File: src/components/Demo.vue [hunk 2]
```diff
@@ -59,9 +61,7 @@ export default {
     },
 
       methods:{
-         getImage(imgUrl) {
-            return require('../../src/assets/'+imgUrl);
-         }
+         getImage
```
[the remark this answers is not in the document]
Comment (MGmadhavan) [reply src/components/Demo.vue:64]: resolved

File: src/components/ErrorMsgs.vue [hunk 3]
```diff
@@ -25,7 +28,10 @@
       </tbody>
    </table>
 </div>
-</template>
+
+</section>
+</template>>
```
[the remark this answers is not in the document]
Comment (MGmadhavan) [reply src/components/ErrorMsgs.vue:33]: resolved

File: src/components/Demo.vue (hunk 1 above)
Comment (MGmadhavan) [reply src/components/Demo.vue:22]: resolved

**Review by prasannasurya12 — APPROVED** [review]

**Pull request merged**